src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java

```diff
@@ -160,7 +168,9 @@ public boolean isCachedOrg(String orgName) throws Exception {
         
         // query CKAN for the organization information
         String ckanURL = "/api/3/action/organization_show?id=" + orgName;
-        CKANResponse res = requester.doCKANRequest("GET", ckanURL);
+        ArrayList<Header> headers = new ArrayList<Header>();
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5f3e90c696b5b18965d514839a077d242388dd52